blocks.md

```diff
@@ -1332,7 +1336,8 @@ Key | Values | Required | Default
 `interval` | Update interval, in seconds. | No | `600` (10min)
 `warning_threshold` | The threshold of pending (or started) tasks when the block turns into a warning state. | No | `10`
 `critical_threshold` | The threshold of pending (or started) tasks when the block turns into a critical state. | No | `20`
-`filter_tags` | A list of tags a task has to have before its counted as a pending task. | No | ```<empty>```
+`filter_tags` | Deprecated in favour of `filter`. A list of tags a task has to have before its counted as a pending task. The list of tags will be appended to the base filter `-COMPLETED -DELETED`. | No | ```<empty>```
```

Comment:
I'm guessing you meant to say `filters` here?

Reply:
Good catch - fixed!